packages/flutter_localizations/lib/src/l10n/cupertino_pt.arb

```diff
@@ -18,10 +18,10 @@
   "cutButtonLabel": "Cortar",
   "copyButtonLabel": "Copiar",
   "pasteButtonLabel": "Colar",
-  "selectAllButtonLabel": "Selecionar tudo",
+  "selectAllButtonLabel": "Selecionar Tudo",
```

Comment:
"tudo" shouldn't be capitalised here.

Reply:
I'll address this with https://github.com/flutter/flutter/issues/127871.